gr-blocks/lib/add_const_cc_impl.cc

```diff
@@ -43,30 +43,28 @@ add_const_cc_impl::add_const_cc_impl(gr_complex k)
 {
 }
 
+void add_const_cc_impl::set_k(gr_complex k)
+{
+    gr::thread::scoped_lock guard(d_setlock);
+    d_k = k;
+    std::fill(d_k_copy.begin(), d_k_copy.end(), d_k);
+}
+
 int add_const_cc_impl::work(int noutput_items,
                             gr_vector_const_void_star& input_items,
                             gr_vector_void_star& output_items)
 {
     const gr_complex* iptr = (const gr_complex*)input_items[0];
     gr_complex* optr = (gr_complex*)output_items[0];
 
-    int size = noutput_items;
+    unsigned long input_size = static_cast<unsigned long> (noutput_items);
```

Comment:
This looks like it doesn't agree with our clang-format.

Reply:
@mbr0wn sorry, I didn't realize that. I hadn't seen in the guides to use git-clang-format. I'll push another commit to fix these unless you say otherwise. Thanks.

Reply:
That would be great. It avoids us from having to run clang-format outside of other changes, which doesn't help with cherry-pickability and  bisecting.

Reply:
@mbr0wn okay, these are fixed in the most recent commit. Thanks!